### ISSUE TITLE
Get at the cache contents

### ISSUE THREAD

**Issue by adamwolf** [mborsetti/webchanges#27]
As i develop jobs I find myself wanting to check that my filter rules aren't too egregious.  I want to make sure webchanges is seeing the content I'm looking for.

I tried the textfiles storage option, but the naming of the files is not really meant for humans :). I used some sqlite3 commands to print the cache, and that works ok, but... is this a common enough thing that it could use a CLI option?

Am I thinking about this wrong? Am i missing a better way to do this?

Thanks!  webchanges seems pretty awesome.

**Comment by mborsetti** [comment]
Yes (I think): use [--test](https://webchanges.readthedocs.io/en/stable/cli.html#test-run-a-job) and [--test-diff](https://webchanges.readthedocs.io/en/stable/cli.html#show-diff-from-saved-snapshots).  If those don't work, then I am not sure what you're trying to achieve: `--test` shows exactly what would be saved in the database ("seeing the content I'm looking for") and `--test-diff` shows you the result any filters applied to diffing, i.e. exactly what would be reported.
